integration_test/scenarios/chat_scenario.dart

```diff
@@ -481,4 +482,38 @@ class ChatScenario extends BaseScenario {
     s.softAssertEquals( await CoreRobot($).isActuallyScrollable($,root: $(SingleChildScrollView),), true,
         'Chat list is not scrollable',);
   }
+
+  bool isPinAChat(TwakeListItemRobot takeListItem) {
+    final pin = takeListItem.getPinIcon();
+    return pin.visible;
+  }
+  
+  Future<void> pinAChat(String title) async {
+    final twakeListItem = ChatListRobot($).getChatGroupByTitle(title);
```

Comment:
how about if this group chat not visible in chat list? need to scroll to find or sth else?